app/views/comment_mailer/comment_added_notification.html.haml

```diff
@@ -9,7 +9,7 @@
 - cta_subheader = t("mailer.updated_subheader_html",
   public_id: @proposal.public_id,
   requester_name: @proposal.requester.full_name,
-  proposal_status: @proposal.status)
+  proposal_status: @proposal.status.capitalize)
```

Comment:
we may want to use `display_status` here rather than `status` (`display_status` is a decorated method that says `pending approval` or `pending purchase` depending on the next step if a proposal is pending)